platform/stateful-application/ansible/inventory/junior-h/.inventory_aws_ec2.yml

```diff
@@ -3,7 +3,7 @@ plugin: aws_ec2
 boto_profile: junior-h
 
 regions:
-  - us-east-1
+  - AWS_REGION_TO_REPLACE
```

Comment:
Is there a way we can use env vars instead?
maybe create a inventory_aws_ec2.yml.template?
the name is inventory/junior-h/.inventory_aws_ec2.yml, I think has an extra dot before inventory_aws_ec2.yml we could remove it

Reply:
1) The aws_ec2 plugin doesn't allow overwriting the [region](https://docs.ansible.com/ansible/latest/collections/amazon/aws/aws_ec2_inventory.html#parameter-regions) using environment variables.

2) I had originally named the file `inventory_aws_ec2.yml-template` as it would be much more intuitive. The problem was that Ansible tried use it as a configuration file. It would complain (4 or 5 files of warnings and errors) about that file not being valid and then it continued with the file generated with sed. I chose to use a hidden file to avoid this nuance.